src/utils/Validate/validateRequired.js

```diff
@@ -16,6 +16,14 @@ const validateRequired = (value, label = '') => {
     }
   }
   if (!hasValue) {
+    if(customErrors && Array.isArray(customErrors)){
+      const result = customErrors.filter(error => {
+        return error.type === 'required';
+      })
+      if(result && result.length > 0 && result[0].message){
```

Comment:
Purely aesthetically...
```suggestion
      if (result && result.length > 0 && result[0].message) {
```